### ISSUE TITLE
Rename hoogle.haskell.org URL

### ISSUE THREAD

**Issue by adius** [ndmitchell/hoogle#242]
I just accidentally landed on and was really confused why the results are so bad.
Until (after way too much googling) I realized I that hoogle.haskell.org is some kind of alpha version
and haskell.org/hoogle is the real deal.

Can you please rename this URL or remove it entirely?
It doesn't feel very Haskell like to give me URLs without type safety 😂.

**Comment by ndmitchell** [comment]
The new `hoogle.haskell.org` is much better in many ways, and the plan is to switch over to it. Could you detail some of the queries you tried that gave you much worse results? I assume all type-based searches?

**Comment by adius** [comment]
Ah OK ^^

Yes, I guess they were all type based

**Comment by ndmitchell** [comment]
Concrete examples of what you tried would be handy as then I can focus on fixing those first.

**Comment by adius** [comment]
Just really simple things like `String -> Int`.
Only the last entry is really `String -> Int` all the others are `Int -> String` 🙈

**Comment by ndmitchell** [comment]
OK, that's probably fairly simple to fix.

**Comment by adius** [comment]
(However, the issue still stands. Then the haskell.org/hoogle URL should be renamed / redirected.)

**Comment by adius** [comment]
It's just crazy that they look the same and have equally official URLs.

**Comment by ndmitchell** [comment]
Yep, agreed - redirecting the old to the new is something on my todo list! Unfortunately a few jobs got in the way so the situation has persisted years longer than I hoped.
